android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/Addon.kt

```diff
@@ -291,8 +287,16 @@ data class Addon(
             val permissions = extension.getMetadata()?.permissions.orEmpty() +
                 extension.getMetadata()?.hostPermissions.orEmpty()
 
+            val developerName = extension.getMetadata()?.developerName.orEmpty()
+            val author = if (developerName.isNotBlank()) {
+                Author(name = developerName, url = extension.getMetadata()?.developerUrl.orEmpty())
+            } else {
+                null
+            }
+
```

Comment:
Could we add a test case to valid this new behaviour in [AddonTest.k](https://github.com/mozilla-mobile/firefox-android/blob/c939c369cf05ce96a5fb34ecd8893f58381933da/android-components/components/feature/addons/src/test/java/AddonTest.kt#L359) ? :)